src/server.ts

```diff
@@ -243,7 +247,7 @@ export class Server {
     );
 
     return HorizonAxiosClient.post(
-      URI(this.serverURL)
+      URI(this.serverURL.toString())  // TODO: Is toString() neccessary?)
```

Comment:
The other calls to URI don't do toString() so let's get rid of it!

Reply:
good catch